src/components/hp-mixed-reality-controls.js

```diff
@@ -0,0 +1,191 @@
+var bind = require('../utils/bind');
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var trackedControlsUtils = require('../utils/tracked-controls');
+var checkControllerPresentAndSetup = trackedControlsUtils.checkControllerPresentAndSetup;
+var emitIfAxesChanged = trackedControlsUtils.emitIfAxesChanged;
+var onButtonEvent = trackedControlsUtils.onButtonEvent;
+
+// See Profiles Registry:
+// https://github.com/immersive-web/webxr-input-profiles/tree/master/packages/registry
+// TODO: Add a more robust system for deriving gamepad name.
+var GAMEPAD_ID_PREFIX = 'hp';
+var GAMEPAD_ID_SUFFIX = '-mixed-reality';
+var GAMEPAD_ID_COMPOSITE = GAMEPAD_ID_PREFIX + GAMEPAD_ID_SUFFIX;
```

Comment:
Perhaps we don't need to have suffix / prefix on this one and can have just `var GAMEPAD_ID`

Reply:
Collapsed this into `GAMEPAD_ID`